lib/AST/StmtPrinter.cpp

```diff
@@ -1735,6 +1735,46 @@ void StmtPrinter::VisitAtomicExpr(AtomicExpr *Node) {
   OS << ")";
 }
 
+// Checked C extension
+
+void StmtPrinter::VisitCountBoundsExpr(CountBoundsExpr *Node) {
+  switch (Node->getKind()) {
+    case CountBoundsExpr::Kind::Byte_Count:
+      OS << "byte_count(";
+    case CountBoundsExpr::Kind::Count:
+      OS << "count(";
+    default:
+      llvm_unreachable("count bounds kind not covered by switch");
+      break;
```

Comment:
Are these breaks under llvm_unreachable calls redundant?  If llvm_unreachable is marked as noreturn, then it's probably safe to elide the breaks.